lib/msal-browser/docs/login-user.md

```diff
@@ -68,12 +68,19 @@ When a login call has succeeded, you can use the `getAllAccounts()` function to
 const myAccounts: AccountInfo[] = msalInstance.getAllAccounts();
 ```
 
-If you know the username of the account, you can also retrieve the account information by using the the `getAccountByUsername()` API:
+If you know the account information, you can also retrieve the account information by using the the `getAccountByUsername()` and `getAccountById()` APIs:
```

Comment:
```suggestion
If you know the account information, you can also retrieve the account information by using the `getAccountByUsername()` or `getAccountById()` APIs:
```

Reply:
Since `getAccountById()` is the reliable call in all cases, can we please rephrase this as: If the user has the account information, they can also retrieve the account information by calling the API:  `getAccountById()`, if the user is certain that the account has a `upn` claim, they can retrieve the account information with the API: ``getAccountByUsername()`. Please note for B2C, this is what is expected when you make this call .... << add details here or a link to the B2C docs once they are created>>